includes/blast_ui.admin.inc

```diff
@@ -156,6 +164,100 @@ function blast_ui_admin_form_validate($form, &$form_state) {
 
 }
 
+
+/*
+**
+** custom method disabled the web services 
+**
+*/
+function disable_web_services()
+{
+    $ar=array('services', 'rest_server');
+    $result = module_disable($ar,true);
+    try{
+    $eid = 1;
+    $num_deleted = db_delete('services_endpoint')->condition('eid', $eid)->execute();  
+    } catch(Exception $e){
+    //drupal_set_message(t("Web-Services of blast are already disabled"), 'error');
+   }
+  
+}
+
+
+/*
+** custom method enable the web services 
+**
+*/
+function enable_web_services()
+{
+  $ar=array('services', 'rest_server');
+  $result = module_enable($ar, true);
+  if (empty($result)) {
+   throw new DrupalUpdateException(implode(', ', $ar) . ' or dependencies could not be enabled, please install services ');
+  }
+   
+  $server_settings = array(
+    'formatters' => array(
+      'bencode' => TRUE,
+      'json' => TRUE,
+      'jsonp' => TRUE,
+      'php' => TRUE,
+      'xml' => TRUE,
+    ),
+    'parsers' => array(
+      'application/json' => TRUE,
+      'application/vnd.php.serialized' => TRUE,
+      'application/x-www-form-urlencoded' => TRUE,
+      'application/xml' => TRUE,
+      'multipart/form-data' => TRUE,
+      'text/xml' => TRUE,
+    ),
+  );
+
+  $resources = array(
+    'blast' => array(
+      'operations' => array(
+        'retrieve' => array(
+          'enabled' => '1',
+        ),
+        'index' => array(
+          'enabled' => '1',
+        ),
+      ),
+      'actions' => array(
+        'makeJobRequest' => array(
+          'enabled' => '1',
+        ),
+        'getBlastStatus' => array(
+          'enabled' => '1',
+        ),
+        'getDatabaseOptions' => array(
+          'enabled' => '1',
+        ),
+      ),
+    ),
+  );
+
+  try{
+    $nid = db_insert('services_endpoint')->fields(
+    array(
+      'eid' => 1,
```

Comment:
Is it valid to set this directly? What happens if there is already a service endpoint with the eid?